src/components/KonnectorList.jsx

```diff
@@ -7,13 +7,18 @@ const VOTING_LINK = 'https://framaforms.org/cozy-collect-1494574386'
 
 const KonnectorList = ({ t, konnectors, showVoting = false }) => (
   <div className="connector-list">
-    {konnectors.map(konnector => (
-      <KonnectorTile
-        konnector={konnector}
-        subtitle={t(`category.${konnector.category}`)}
-        route={`${konnector.slug}`}
-      />
-    ))}
+    {konnectors.map(konnector => {
+      const categories = konnector.categories.map(category =>
```

Comment:
Same remark here than in my next comment (sorry it is not usual).

Also, maybe we could have a konnector helper  wich may provide a `translateCategories(konnector)` method:
```js
const translateCategories = (konnector, t) => konnector.categories && konnector.categories.map(c => t(`category.${c}`)).join(', ') || ''
```
What do you think ?